gruntfile.js

```diff
@@ -277,6 +285,8 @@ module.exports = function (grunt) {
   grunt.registerTask('test', ['env:test', 'lint', 'mkdir:upload', 'copy:localConfig', 'server', 'mochaTest', 'karma:unit']);
   grunt.registerTask('test:server', ['env:test', 'lint', 'server', 'mochaTest']);
   grunt.registerTask('test:client', ['env:test', 'lint', 'server', 'karma:unit']);
```

Comment:
@mleanos client should really have 2 subfolders, unit and e2e. Both tests test client-side code. If the folder structure really bothers you, feel free to submit a PR because anyway /client is just client-side unit tests and e2e is client-side e2e tests.